test/integration/workflows/DerivativeIntegration.t.sol

```diff
@@ -29,18 +29,17 @@ contract DerivativeIntegration is BaseIntegration {
     function run() public override {
         super.run();
         _beginBroadcast();
-        _logTestStart("DerivativeIntegration");
         _setUpTest();
         _test_mintAndRegisterIpAndMakeDerivative();
         _test_registerIpAndMakeDerivative();
         _test_mintAndRegisterIpAndMakeDerivativeWithLicenseTokens();
         _test_registerIpAndMakeDerivativeWithLicenseTokens();
         _test_multicall_mintAndRegisterIpAndMakeDerivative();
-        _logTestEnd("DerivativeIntegration");
         _endBroadcast();
     }
 
     function _test_mintAndRegisterIpAndMakeDerivative() private {
+        _logTestStart("_test_mintAndRegisterIpAndMakeDerivative");
```

Comment:
it would be better if we create a modifier for the logging so that reduce duplicate code like:
modifier log(string memory msg) {
            console2.log(unicode"🏃 Running", testName, "...");
            _
            console2.log(unicode"✅", testName, "passed!");
            
}

Reply:
Resolved in [0eabb60](https://github.com/storyprotocol/protocol-periphery-v1/pull/80/commits/0eabb60a381020c59b71ede766c23fa4cda88f22) :)